B2B-2557 :Add documentation for the new B2B API (#150)
AdobeDocs/commerce-webapi: src/pages/rest/b2b/negotiable-quote.md

> The negotiable quote lifecycle includes a number of stages, as shown on the diagram below.

Author: done

-  The buyer reviews the seller's counteroffer
-  Upon agreement, the buyer begins the checkout process and the system converts the negotiable quote into an order

> Negotiable quote can be also initiated by seller. In this case cycles could be as following

Reviewer: ```suggestion
A negotiable quote can be also initiated by seller. In this case, the lifecycle includes the following stages:
```

- The seller creates and submits a negotiable quote

> - The buyer reviews and modifies the quote

Reviewer 2: ```suggestion
- The buyer reviews and modifies the quote or begins the checkout process
```

Author: Thats description of negotiation cycles. Proposed addition covered in "Upon agreement, the buyer begins the checkout process and the system converts the negotiable quote into an order"

The negotiated price set in the negotiable quote is exactly the price that will be applied on a quote during checkout, order generation, and invoice generation.

Status | Description | Available actions to seller
--- | -- | ---

> Draft | The seller has created an empty quote. Quote is not visible for buyer. | View, submit, save as draft. <br/><br/>Edit name, the expiration date, item quantity, add/remove product items, enter a proposed price, add comments.

Reviewer: ```suggestion
Draft | The seller has created an empty quote that is not visible to the buyer. | View, submit, save as draft. <br/><br/>Edit the name, expiration date, and item quantity, add/remove product items, enter a proposed price, add comments.
```

Author: done

Client reviewed | The buyer has opened the quote submitted by seller and is modifying it, by changing items or adding a shipping address. The seller cannot edit the quote. | View
Updated | The buyer has re-submitted the quote to seller. The seller can edit the quote, but the buyer cannot. | View, submit, decline, save as draft.<br/><br/>Edit the expiration date, item quantity, add/remove product items, enter a proposed price, add shipping method and shipping price, add comments.
Ordered | The buyer has purchased the quote, and Adobe Commerce converts the quote to an order. Neither the seller nor the buyer can edit the quote. | View
Closed | The buyer has cancelled the quote and thus stopped the negotiation process. Neither the seller nor the buyer can edit the quote. <br/><br/>The buyer clicks the **Close** button from the Quote details page. (Not available using Web API) | View
Declined | The seller has declined the quote. All custom pricing (if any) is removed from the quote. In admin panel, the quote is locked for editing.                                                                                                 | View

> Expired | The quote is on the buyer's side, and the quote's expiration date has passed.                                                                                                                                                              | View

Reviewer: ```suggestion
Expired | The expiration date has passed on a quote under buyer review.                                                                                                                                                            | View
```

> Expired | The quote is on the buyer's side, and the quote's expiration date has passed.                                                                                                                                                              | View

Author: done
